src/harnesses/basex/basex-xquery-harness.xproc

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ===================================================================== -->
+<!--  File:       basex-xquery-harness.xproc                               -->
+<!--  Author:     Florent Georges                                          -->
+<!--  Date:       2011-08-30                                               -->
+<!--  URI:        http://xspec.googlecode.com/                             -->
+<!--  Tags:                                                                -->
+<!--    Copyright (c) 2011 Florent Georges (see end of file.)              -->
+<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
+<p:pipeline xmlns:p="http://www.w3.org/ns/xproc" xmlns:c="http://www.w3.org/ns/xproc-step" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:t="http://www.jenitennison.com/xslt/xspec" xmlns:pkg="http://expath.org/ns/pkg" pkg:import-uri="http://www.jenitennison.com/xslt/xspec/basex/harness/xquery.xproc" name="basex-xquery-harness" type="t:basex-xquery-harness" version="1.0">
+
+	<p:documentation>
+		<p>This pipeline executes an XSpec test suite with BaseX standalone.</p>
+		<p><b>Primary input:</b> A XSpec test suite document.</p>
+		<p><b>Primary output:</b> A formatted HTML XSpec report.</p>
+		<p>The dir where you unzipped the XSpec archive on your filesystem is passed
+			in the option 'xspec-home'.  The compiled test suite (the XQuery file to be
+			actually evaluated) is saved on the filesystem to be passed to BaseX.  The
+			name of this file is passed in the option 'compiled-file' (it defaults to a
+			file in /tmp).</p>
+	</p:documentation>
+
+	<p:serialization port="result" indent="true" method="xhtml" encoding="UTF-8" include-content-type="true"/>
+
+	<p:option name="xspec-home" required="true"/>
+	<p:option name="basex-jar" required="true"/>
+	<!-- TODO: Use a robust way to get a tmp file name from the OS... -->
+	<p:option name="compiled-file" select="'file:/tmp/xspec-basex-compiled-suite.xq'"/>
+	<!-- TODO: Use the absolute URIs through the EXPath Packaging System. -->
+
+	<p:variable name="compiler" select="        resolve-uri('src/compiler/generate-query-tests.xsl', $xspec-home)"/>
```

Comment:
Just a nitpick. 😺 
You seem to have reformatted the XML. So now there are extra spaces, which looks a bit odd. (Spaces between `select="` and `resolve-uri`)